aws/resource_aws_dx_gateway_association.go

```diff
@@ -12,30 +13,47 @@ import (
 )
 
 const (
-	GatewayAssociationStateDeleted = "deleted"
+	gatewayAssociationStateDeleted = "deleted"
 )
 
 func resourceAwsDxGatewayAssociation() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceAwsDxGatewayAssociationCreate,
 		Read:   resourceAwsDxGatewayAssociationRead,
+		Update: resourceAwsDxGatewayAssociationUpdate,
 		Delete: resourceAwsDxGatewayAssociationDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceAwsDxGatewayAssociationImport,
+		},
 
 		Schema: map[string]*schema.Schema{
+			"allowed_prefixes": {
```

Comment:
I think it is okay to mark this as `Computed: true` for now, given that it automatically includes the VPC CIDR if no prefixes are provided. 👍

Reply:
Done, I think a missing call to `resourceAwsDxGatewayAssociationRead()` at the end of `resourceAwsDxGatewayAssociationCreate()` doesn't help either 😄.

Reply:
```console
$ make testacc TEST=./aws/ TESTARGS='-run=TestAccAwsDxGatewayAssociation_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws/ -v -parallel 20 -run=TestAccAwsDxGatewayAssociation_basic -timeout 120m
=== RUN   TestAccAwsDxGatewayAssociation_basic
=== PAUSE TestAccAwsDxGatewayAssociation_basic
=== CONT  TestAccAwsDxGatewayAssociation_basic
--- PASS: TestAccAwsDxGatewayAssociation_basic (943.05s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	943.077s
```